src/plugins/discover/public/application/main/components/top_nav/get_top_nav_badges.tsx

```diff
@@ -53,5 +59,17 @@ export const getTopNavBadges = ({
     });
   }
 
+  if (isManaged) {
+    entries.push({
+      data: getManagedContentBadge(
+        i18n.translate('discover.topNav.managedContentLabel', {
+          defaultMessage:
+            'This saved search is managed by Elastic. Changes here must be saved to a new saved search.',
+        })
+      ),
+      order: -100,
+    });
+  }
```

Comment:
Minor UX nit/question: I wonder if it would be better to always make this the rightmost badge instead to prevent it jumping around when reverting changes?
![badge](https://github.com/elastic/kibana/assets/25592674/4d9759ac-8631-427a-8416-7556a635e029)

Reply:
Makes sense to me!

Reply:
Done in https://github.com/elastic/kibana/pull/175256/commits/74d2a5ab3eba216f997fdb6a122fba39208d7ad7